Sources/BuildServerIntegration/LegacyBuildServer.swift

```diff
@@ -20,8 +20,8 @@ import SKOptions
 import SwiftExtensions
 import ToolchainRegistry
 
-#if compiler(>=6.3)
-#warning("We have had a one year transition period to the pull based build server. Consider removing this build server")
+#if compiler(>=6.5)
+#warning("We have had a two year transition period to the pull based build server. Consider removing this build server")
```

Comment:
https://github.com/SolaWing/xcode-build-server is the only BSP server that I am aware of, which uses the legacy push-based model. While it has transitioned to the pull-based model (https://github.com/SolaWing/xcode-build-server/commit/a030fea71b0701a30217d5ab5eb8cfb56395c0a0), it has not released a version that includes this change. Add another year of transition period to allow all BSP servers to transition.